index.rst

```diff
@@ -6,24 +6,119 @@
+
+.. image:: ./_static/Bump_Test_Results.png
+
+Figure 4. This shows the actual measured forces applied by the actuators during the bump test of a single actuator
+
+
+Bump Test Indexing
+==================================
+
+The actuators are numbered sequentially from 0-155.  The secondary tests and the X, Y, and Z forces are also numbered sequentially.  Since there are different numbers of each of these, a set of look-up tables are provided to go from the actuator ID to the sequential numbers of the actuators of different types.  Table 1 shows how to move between the actuator IDs, the actuator index, and the indices for the various types of actuators. 
+
+
+| import lsst.ts.cRIOpy.M1M3FATable as M1M3FATable
+| FATABLE = M1M3FATable.FATABLE
+|
+| Actuator ID (as in Figure 1): 227
+| ID to index:  M1M3FATable.actuatorIDToIndex(227) : 61
+| index to ID: FATABLE[index][M1M3FATable.FATABLE_ID]: 227
+| Secondary index: FATABLE[index][M1M3FATable.FATABLE_SINDEX]: 46
+| Y index: FATABLE[index][M1M3FATable.FATABLE_YINDEX]: 42
+| Z index: FATABLE[index][M1M3FATable.FATABLE_ZINDEX]: 61
+|
+| Since all actuators provide force in the Z direction, the ZINDEX and the INDEX are equal.
+|
+| The following actuator is one of the few that provides force in the X direction.
+|
+| Actuator ID (as in Figure 1): 135
+| ID to index:  M1M3FATable.actuatorIDToIndex(135) : 34
+| X index: FATABLE[index][M1M3FATable.FATABLE_XINDEX]: 2
+
+
+Table1: Navigating between actuator IDs and sequential indices.
+
+Bump Test States
+==================================
+
+As the bump test cycles through the testing, the EFD location:
+
+| lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus
+|
+
+returns where in the bump test cycle the test is.  Figure 5 shows the progression of these states.  At the conclusion of the test for each actuator, the state will either be 6 (PASSED) or 7 (FAILED).
+
+.. image:: ./_static/Bump_Test_States.png
+
+Figure 5. This shows the bump test states as logged in  lsst.sal.MTM1M3.logevent_forceActuatorBumpTestStatus.
+
+XYZ forces vs Cylinder forces
+==============================
+
+The force data in the EFD includes both the forces applied to the primary and secondary cylinders, as well as the forces in the X, Y, and Z directions.  These forces are not the same, as will be explained in this section.  Referring to Figure 6, we see that the primary cyclinder applies force in the axial direction, which is along the optical axis of the telescope, and is referred to as the Z direction.  So the force applied by the primary cylinder is equal to the Z-force.  The secondary cylinder however, is at a 45 degree angle to the Z-axis.  So if we want to apply a force in the lateral direction (X or Y), we need to apply a force with the secondary cylinder, and then a negative force with the primary cylinder so that the resultant is in the lateral direction.
```

Comment:
in the mirror coordinate system - X, Y and Z directions. (there should be a link on how the mirror coordinate system is defined - perhaps this? https://confluence.lsstcorp.org/pages/viewpage.action?pageId=47220348 or that https://confluence.lsstcorp.org/display/LTS/Subsystem+Coordinate+Transformation or both)